crates/cxx-qt-lib/src/core/qmap/mod.rs

```diff
@@ -50,6 +50,24 @@ where
     }
 }
 
+impl<T> PartialEq for QMap<T>
+where
+    T: QMapPair,
+    T::Value: PartialEq,
+{
+    /// Returns true if both maps contain the same key value pairs
+    fn eq(&self, other: &Self) -> bool {
+        self.len() == other.len() && self.iter().all(|(k, v)| &T::value(other, k) == v)
```

Comment:
Ah was probably missed in the `value_or_default` fixes that were being done in one of the other branches, I'll fix it next week 👍

Reply:
In any case it might be a good idea to additionally implement something like [HashMap::get()](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.get) that returns an `Option` in addition to the `value_or_default()`, as I think this is much more idiomatic than default constructing some random value. I could do that as part of this PR.

Reply:
Looks like it was `QVariant` that has the `or_default` parts, we should copy that for the associated container types too and have `value(&self, &Self::K) -> Option<T>` and `value_or_default(&self, &Self::K) -> T`. QList/QVector already have an Option.

Reply:
Then there is also the discussion over whether the naming should follow the Qt name (value) or the Rust name (get), probably the latter :-)

Reply:
I can fix this in a separate pull request :-)

Reply:
Split out here #443